src/Resource/AbstractResourceList.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PCF\DisposableEmail\Resource;
+
+/**
+ * Class AbstractResourceList
+ * @package PCF\DisposableEmail\Resource
+ */
+abstract class AbstractResourceList
+{
+    /**
+     * @var string
+     */
+    protected static $list = '';
+
+    /**
+     * @return array
+     */
+    public static function getList(): array
+    {
+        return explode(PHP_EOL, trim(static::$list));
```

Comment:
I think about it and yes i will clone source files as original. Imo this is not reasonable to be afraid about perfomance and efficient in this case. Users and frameworks can cache this data in faster storages than static txt. 

Current frameworks have fine mechanism for caching static resources (ex. routing)for example [syfony](http://symfony.com/doc/current/components/dependency_injection/compilation.html)